### ISSUE TITLE
Some spaces and clothing are missing 

### ISSUE THREAD

**Issue by Tiff2serious** [DestinationHome/Destination-Home-Online#128]
**Describe the bug**
When I look on the screen spaces that I had in past beta release are missing also clothing. I know you're still working on things, just wanted to point it out.

**To Reproduce**

**Expected behavior**

**Screenshots & Video Documentation**
If applicable, add screenshots or videos to help explain your problem. This will allow the Destination Home team to fix critical bugs faster for our community. 

# Client Information:
**Region:** [SCEA]
**Country:** [North America] 
**Platform:** [PS3]
**Firmware Type:** [HFW]

**Additional context**
Add any other context about the problem here, that you may see fit to give a general overview of this issue.

**Comment by MrJunez** [comment]
Have you tried clearing cache and see if those certain items or spaces come back after it is cleared?

**Comment by Tiff2serious** [comment]
I will  do so thanks

**Comment by Tiff2serious** [comment]
> Have you tried clearing cache and see if those certain items or spaces come back after it is cleared?

Yes I did it yesterday and some are still missing.

**Comment by Tiff2serious** [comment]
> Have you tried clearing cache and see if those certain items or spaces come back after it is cleared?

Yes I tried it and trying again right now, my Club Luminosity Space, I  had last beta but is not here also the glass house apartment Space I paid for.
